### ISSUE TITLE
Readable Input History

### ISSUE THREAD

**Issue by ejrbuss** [awiggs/chip-gr8#40]
Do not start this issue until `inputHistory` is fully implemented.

Input history will likely not be very readable as it will contain inputs as keymasks. To make this output easier to understand it would be nice to have a utility function that takes an input history and a set of labelled game actions and returns a human readable interpretation of this history.
